storage/cleanup.go

```diff
@@ -89,7 +89,12 @@ func (m *cleanupManager) Cleanup(t time.Time) error {
 			"encountered errors when cleaning up fileset files for %v: %v", t, err))
 	}
 
-	commitLogStart, commitLogTimes := m.commitLogTimes(t)
+	commitLogStart, commitLogTimes, err := m.commitLogTimes(t)
+	if err != nil {
+		multiErr = multiErr.Add(fmt.Errorf(
+			"encountered errors when cleaning up commit logs: %v", err))
```

Comment:
i'd suggest returning `multiErr.FinalError()` after this line

Reply:
yeah that makes sense